tests/python/frontend/tflite/test_forward.py

```diff
@@ -793,13 +799,14 @@ def _test_elemwise(math_op, data, fused_activation_function=None, quantized=Fals
 
         if quantized:
             inq_const = tf.quantization.fake_quant_with_min_max_args(data[0], min=-100, max=100, name="const_tensor")
-            inq_data = [tf.quantization.fake_quant_with_min_max_args(in_data[0], min=-100, max=100, name="inq_1")]
+            inq_data = [tf.quantization.fake_quant_with_min_max_args(in_data[0], min=-50, max=50, name="inq_1")]
+            input_range = {'inq_1': (-100, 100)}
```

Comment:
I'm a little confused here. I think the `input_range` of `inq_1` is (-50,50)? Because you modify the range of `inq_data`.

Reply:
Thanks for spotting the mistake, I will fix it